pom.xml

```diff
@@ -35,6 +35,7 @@
     <folio-spring-base.version>7.2.0</folio-spring-base.version>
     <folio-spring-cql.version>7.2.0</folio-spring-cql.version>
     <folio-service-tools.version>3.1.0</folio-service-tools.version>
+    <folio-query-tool-metadata.version>1.1.0-SNAPSHOT</folio-query-tool-metadata.version>
```

Comment:
Please, change 1.1.0-SNAPSHOT version to released one https://github.com/folio-org/folio-query-tool-metadata/releases/tag/v1.0.0.

Reply:
done